### PR TITLE
Mobile version

### PR DESCRIPTION
**Pull request by PNdunguMaina**
In this project, I did the following:

- created an index.html file in VSCode and placed HTML elements on the webpage for the home page and about page.
- Used Javascript to create elements dynamically for the featured artists' section.
- created a mobile menu that pops up when the user clicks the hamburger button.
- created a responsive website using media queries.
- added styles to the HTML elements created to make the website appealing

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (Lordkaito) [inline index.html:2]: - [ ] I can see in your project that you built only the home page for the capstone, but you are supposed to build both about and homepage, so please make sure to include the about page. Here is an image of what the homepage and the about page look like: 
- Homepage
main

- About page
about

File: index.html [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+
+      <div id="copyright" class="copyright desktop">
+        <i class="fa-solid fa-copyright">Concerts Arena</i>
+        <div class="copyright-text">
+          <p>2022 Concerts Arena. Some rights reserved
+        </p>
+        </div>
+      </div>
+
+    </footer>
```
Comment (Lordkaito) [inline index.html]: - [ ] Regarding your footer, please make sure that it follows the original design as much as possible. Right now you barely have a footer and also, in the original design, it is supposed to have a dark grey background color, as well as it is supposed to have some more height and the text should be centered. Please work on your design.

File: script.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
+const artists = [
```
Comment (Lordkaito) [inline script.js:1]: - [ ] If you take a look at your artist's section on the mobile version, you will realize that the text is overlapping the images of the artists, here's an image: 
Captura de pantalla 2022-06-30 165243

So please make sure to fix this issue.

**Review by Lordkaito — CHANGES_REQUESTED** [review]
Hi @PNdunguMaina, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by arslanbisharat — APPROVED** [review]
Hi @PNdunguMaina ,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**